client/src/components/totalReviews.jsx

```diff
@@ -1,16 +1,33 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+
+import StarRatings from 'react-star-ratings';
+
 
 export default class TotalReviews extends React.Component {
   constructor(props) {
     super(props);
   }
 
   render() {
-    return (
-      <div>
-
-      </div>
-    );
+    if (this.props.reviews.length) {
+      return (
+        <div id="totalRevsContainer">
+          <div>{this.props.reviews.length} Reviews</div>
+          <div>
+            <StarRatings
```

Comment:
If you decide to create a "Stars" component, this would be another good place to reuse